### PR TITLE
New attribute JUnit reporter - target

### PR DESCRIPTION
**Pull request by piotrgo**
Append target for JUnit reporter, so that it's easier to find which host actually failed, when running the same set of controls across multiple nodes.

Fixes https://github.com/chef/inspec/issues/2840

### CODE REVIEW

**Review by jquick — APPROVED** [review]
Looks good to me! Nice addition, thanks @piotrgo.

**Review by jerryaldrichiii — APPROVED** [review]
Great addition @piotrgo! Also, congratulations on your first contribution to InSpec. 😄

**Review by jerryaldrichiii — DISMISSED** [review]
@piotrgo, apologies for not catching this earlier. Before we can merge this you'll need to perform a DCO signoff.

See: https://github.com/chef/inspec/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

**Comment by jquick** [comment]
You should be able to amend and sign off by:
```
git commit -s --amend
git push -f
```

**Comment by chris-rock** [comment]
@piotrgo Thank you very much for this PR. I would like to hold-off on merge right now until we have defined exactly which fields we are going to add to the report. There is nothing wrong with this PR, we just need to have a consistent approach for all reporters.

Since we are adding a automate reporter soon too, we are going to add additional fields as well like the node uuid. Can we define first, which fields we are going to add?

**Comment by piotrgo** [comment]
Sure thing, I wasn't aware of the broader action, although for CLI reporter node under test is present, so I thought it's been simply missed for JUnit.

**Comment by chris-rock** [comment]
We just started working on it ;-) Thank you for your patience. I fully understand your use case but we should really strive for consistency behavior across reporters if possible. We let you know once we docuemted the current idea.

**Review by chris-rock — APPROVED** [review]
Thank you again @piotrgo for your patience. Your PR works perfectly with our approach! So we do not need to change anything.

**Pull request merged**
